spyder/plugins/editor/widgets/completion.py

```diff
@@ -22,7 +22,7 @@
 from spyder.widgets.helperwidgets import HTMLDelegate
 
 
-COMPLETION_ITEM_TEMPLATE = """
+COMPLETION_ITEM_TEMPLATE = u"""
```

Comment:
Either we add this `u` or `unicode_literals` import on top.

Any preference @ccordoba12 ? (this was failing for me on python 2)

Reply:
Using `u` is fine. `unicode_literals` can introduce undesired side effects, that's why we don't use it.